File: vimeovideo_info.py
import urllib2
import simplejson
 
videoid = "52387087"
url = "http://vimeo.com/api/v2/video/" + videoid + ".json"
handle = urllib2.urlopen(url)
content = handle.read()
j = simplejson.loads(content)
 
for key in j[0]:
    print "{0}: {1}".format(key,j[0][key])

File: vimeovideo_info.php
$id = "52387087";
$url = "http://vimeo.com/api/v2/video/". $id .".json";
$j = json_decode(file_get_contents($url));
 
foreach($j[0] as $key => $value) {
   echo $key . ": " . $value . "n<br />";
}
